### ISSUE TITLE
Webhook initial "ping" request sends a webhook ID of `0`

### ISSUE THREAD

**Issue by thomasplevy** [gocodebox/lifterlms-rest#150]
## Reproduction Steps

1) Create a new webhook for any topic
2) Inspect the received "ping" event body

## Expected

The body should be something like:

```json
{
  "webhook_id": 1234,
}
````

where "1234" equals is the resource ID of the newly created webhook

## Actual

The received event body will always have an ID of "0":

```json
{
  "webhook_id": "0",
}
````

## Information

I believe this is happening because the hookshot is fired before the webhook database object is created so there is no ID to send.

**Comment by eri-trabiccolo** [comment]
mmm but this means that we have to save the webhook before pinging...
while at the moment the save is subordinated to the pinging success.

**Comment by thomasplevy** [comment]
Yes, it does...  I'm not sure the best way to resolve but this is something of a minor issue.

We might have to save without pinging and change the state immediately if the ping fails.
